src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/FindNodeVisitor.cs

```diff
@@ -40,7 +40,7 @@ public override bool PreVisit(VariadicOpNode node)
             Contracts.AssertValue(node);
             Contracts.Assert(node.Children.Count > 0);
 
-            var numTokens = CollectionUtils.Size(node.OpTokens);
+            var numTokens = CollectionUtils.Size(node.OpTokens.ToArray());
```

Comment:
>ToArray [](http://example.com/codeflow?start=63&length=7)

extra ToArrays() might be an issue here. 

Especially since we're just getting the size. 
Could we add an overload of CollectionUtils.Size and avoi the .ToArray()?